src/integrationTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/AttachExceptionRecordToExistingCaseTest.java

```diff
@@ -535,6 +534,17 @@ private void verifyRequestedAttachingToCase() {
         );
     }
 
+    private void verifySuccessResponseWithAttachToCaseReference(ValidatableResponse response) {
+        JsonPath responseJson = response.extract().jsonPath();
+
+        assertThat(responseJson.getList(RESPONSE_FIELD_ERRORS)).isNullOrEmpty();
+        assertThat(responseJson.getList(RESPONSE_FIELD_WARNINGS)).isNullOrEmpty();
+
+        Map<String, Object> responseData = responseJson.getMap(RESPONSE_FIELD_DATA);
+        assertThat(responseData).isNotNull();
+        assertThat(responseData.get("attachToCaseReference")).isEqualTo(CASE_REF);
```

Comment:
maybe `CASE_REF` should be a parameter?

Reply:
As part of this PR I wanted to make case reference a parameter, so that the test would define it. However, `CASE_REF` is so entangled into setup that this is a bigger, and definitely a separate change to apply.